github/Authorization.py

```diff
@@ -27,106 +27,98 @@
 # along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
 #                                                                              #
 ################################################################################
+from datetime import datetime
+from typing import TYPE_CHECKING, List, Optional
 
 import github.AuthorizationApplication
 import github.GithubObject
 
+from .GithubObject import Attribute, NotSet, Opt, _NotSetType
```

Comment:
We have to import `NotSet` and `_NotSetType`. Only using the former would be grand:
instead of
```
assert isinstance(scopes, _NotSetType)
```
we could use
```
assert scopes is NotSet
```
What do you think?

Reply:
we can't, mypy can't narrow type on `scopes is NotSet`